doc/api/cluster.markdown

```diff
@@ -280,21 +624,18 @@ the `'disconnect'` event has not been emitted after some time.
 
 * {Number}
```

Comment:
I'm not sure its a Number always, it looks like a number in the source, but if you get it by doing Object.keys(cluster.workers) its a string. Maybe that just a js-ism not worth mentioning here.